server_admin/topics/identity-broker/first-login-flow.adoc

```diff
@@ -1,94 +1,84 @@
+* This authenticator displays the profile information page, so the user can review their profile {project_name} retrieves from an identity provider.
+* You can set the `Update Profile On First Login` option in the *Actions* menu.
+* When *on*, users are presented with the profile page requesting additional information to federate the user's identities.
+* When *missing*, users are presented with the profile page if the identity provider does not provide mandatory information, such as email, first name, or last name.
+* When *off*, the profile page does not display unless the user clicks in a later phase on the `Review profile info` link in the page displayed by the `Confirm Link Existing Account` authenticator.
 
 Create User If Unique::
-  This authenticator checks if there is already an existing {project_name} account with the same email or username like the account from the identity provider.
-  If it's not, then the authenticator just creates a new local {project_name} account and links it with the identity provider and the whole flow is finished.
-  Otherwise it goes to the next `Handle Existing Account` subflow.
-  If you always want to ensure that there is no duplicated account, you can mark this authenticator as `REQUIRED`. In this case, the user
-  will see the error page if there is an existing {project_name} account and the user will need to link his identity provider account through Account management.
+* This authenticator verifies that there is already an existing {project_name} account with the same email or username as the identity provider's account.
+* If there is not an existing account, the authenticator creates a new local {project_name} account, links this account with the identity provider, and terminates the flow. 
+* If an account exists, the authenticator implements the next `Handle Existing Account` sub-flow.
+* To ensure there is no duplicated account, you can mark this authenticator as `REQUIRED`. The user sees the error page if a {project_name} account already exists, and the user must link their identity provider account through Account management.
 
 Confirm Link Existing Account::
-  On the info page, the user will see that there is an existing {project_name} account with the same email.
-  They can review their profile again and use different email or username (flow is restarted and goes back to `Review Profile` authenticator).
-  Or they can confirm that they want to link their identity provider account with their existing {project_name} account.
-  Disable this authenticator if you don't want users to see this confirmation page, but go straight to linking identity provider account by email verification or re-authentication.
+* On the information page, the user sees an existing {project_name} account with the same email. They can review their profile again and use a different email or username. The flow restarts and goes back to the `Review Profile` authenticator.
```

Comment:
Replace "an existing {project_name} account with the same email. They ...
with
a {project_name} account with the same email. Users"